ethcore/src/extras.rs

```diff
@@ -32,7 +33,9 @@ pub enum ExtrasIndex {
 	/// Block log blooms index
 	BlockLogBlooms = 3,
 	/// Block blooms index
-	BlocksBlooms = 4
+	BlocksBlooms = 4,
+	/// Block receipts index
+	BlockReceipts
```

Comment:
convention is to use trailing `,`s - that way you don't get the issue of having to change a line when appending an entry.